js/views/grid.js

```diff
@@ -97,7 +97,7 @@ DeveloperPlayground.GridView = Backbone.View.extend({
             columns: columns,
             collection: this.collection
             });
-            $("#results").append(grid.render().$el);
+            $("#results").append(grid.render().$el.html());
```

Comment:
I can't really explain why, but this makes table to keep its width.. any thoughts ? @meghanhade @drewda ?